01-data-structures/07-graphs/kevin_bacon.rb

```diff
@@ -0,0 +1,121 @@
+require_relative "Node"
+
+$films = []
+$actors = []
+def find_kevin_bacon(node,start)
+  $actors << start.name
+
+  if start.name == "Kevin Bacon"
+    return $films
+  end
+
+  if $films.include?(:"Footloose")
```

Comment:
Why are you checking specifically for Footloose? What happens if an actor or actress is connected to Bacon through a movie other than Footloose?